requirements-linux-arm64.txt

```diff
@@ -0,0 +1,24 @@
+albumentations==0.4.3
```

Comment:
Why? We have a working 'requirements-mac.txt' with the same content.

The only difference I see here is that you've got a regression in putting 'lstein/k-diffusion' back in here - M* CPUs seem to need 'Birch-san/k-diffusion' to avoid actual bugs

Reply:
This is the only k-diffusion that worked for me on a Linux arm64 container image running on Docker on an arm64 Mac. I'll test again but if I recall Birch-san/k-diffusion did NOT work on linux arm64 container on my M2.

Reply:
In other words this is what works:
macOS arm64 on M2 -> Birch-san/k-diffusion
linux arm64 container on M2 -> lstein/k-diffusion

Reply:
Understood

You can use the existing Mac requirements.txt, and add a pip reinstall after everything else is done:

```
pip install -r requirements-mac.txt
pip install --force-reinstall git+https://github.com/lstein/k-diffusion.git@master#egg=k-diffusion
```

Reply:
Or I might not be completely understanding your goal here. What ML-enabled hardware are you targeting? If it's "CUDA in the cloud", you should definitely be using 'requirements-lin.txt'

Reply:
Although the reason for that specific requirements list is not because I'm on a Mac but because I'm on Linux on arm64 hardware. In my case it happens to be a Mac but I suppose it could have been a Windows arm64 machine running that Linux container, in which case it looks weird to be using requirements-**mac**.txt. The important parts are container OS (Linux) and architecture (arm64).

Reply:
> Or I might not be completely understanding your goal here. What ML-enabled hardware are you targeting? If it's "CUDA in the cloud", you should definitely be using 'requirements-lin.txt', in which case, no changes or workarounds should be required (for requirements).

Yes, that cloud environment would use that requirements file (which could be renamed requirements-linux-**amd**64.txt) but a developer on a Mac would still need to use linux-**arm**64 on their laptop.

Reply:
Now that the project is using Github Actions it might be worth setting up build/test workflows for all these OS/arch combinations. Not all developers will have both amd64 and arm64 machines to test it themselves. But that will have to be a separate PR.